static/sass/_patterns_social-icons.scss

```diff
@@ -4,6 +4,7 @@
     background-image: url('#{$assets-path}6bfc3d6a-build.snapcraft.social.optimize.svg');
     background-repeat: no-repeat;
     background-size: 160px 80px; // 2x4 icons 40x40px each (in 2 rows for regular and hover state)
+    color: transparent;
     display: inline-block;
     height: 40px;
```

Comment:
I'm surprised to see all dimensions hardcoded as px values. This is a problematic practice, especially when we are planning a bump in font-size on bigger screens. All our patterns use rem values which will resolve to different px values once a rem value is changed on a breakpoint.